cube_to_target/cube_to_target.F90

```diff
@@ -407,52 +433,28 @@ program convterr
       !
       ! qqqq: ARH+JTB: double-check please
       !
-      !      call  smooth_intermediate_topo_wrap (rrfac_tmp, da, ncube,nhalo, NSCL_f,NSCL_c, &
-      !           rrfac, terr_dev , &
-      !           smooth_fname, &
-      !           lread_smooth_topofile, &
-      !           luse_multigrid, &
-      !           luse_prefilter, &
-      !           lstop_after_smoothing, &
-      !           lb4b_with_cesm2 , &
-      !           rrfac_tmp )
-      call  smooth_intermediate_topo(terr, da, ncube,nhalo, NSCL_f,NSCL_c, ncube_sph_smooth_iter , & 
```

Comment:
did you remove the smoothing of rrfac? If so, why?

Reply:
I didn't understand the rationale for smoothing rrfac. I interpret rrfac as indicating the the desired dycore resolution at any point on 3000x3000x6 topo grid, so I just use it to change the smoothing radius applied to the topo at i,j when rrfac(i,j)>1.0, but I assume rrfac(i,j) already has the correct value there.  If this doesn't make sense or you disagree we should probably discuss.

Reply:
> but I assume rrfac(i,j) already has the correct value there.

rrfac -even as it is represented in the scrip grid file- is spotty, with anomalous "lagoons" of bad rrfac values. So we found that smoothing after mapping to the 3km grid is required to get rid of all these bad lagoons. 

It's really just to make up for an imperfect rrfac calculation that we use for CAM-SE. I don't know that this will be necessary with MPAS. I also have a new method that greatly improves the rrfac for CAM-SE, but it doesn't work when the grid is rotated (which var-res grids often have rotated grids to align a panel with a region of interest).

Reply:
Alright. Let's discuss this further.  I swas what you meant abot rrfac. I massaged it with the NINT calls before passing to the smoother.